### PR TITLE
FEAT: Added slider component to CL

### PR DESCRIPTION
**Pull request by zoef-yt**
In this pr, I added the Slider component to my  component library

Files to check 
All

As this pr is not merging into the dev branch there is no preview for it.
image

### CODE REVIEW

File: pages/components/slider/slider.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="/styles.css" />
+		<link rel="stylesheet" href="/prism/prism.css" />
+
+		<title>Image | UI Game</title>
```
Comment (sudipkundu999) [inline pages/components/slider/slider.html]: change Title

**Review by sudipkundu999 — APPROVED** [review]
The custom slider looks great.
Good to merge.

File: pages/components/slider/slider.html (hunk 1 above)
Comment (zoef-yt) [reply pages/components/slider/slider.html]: Will do that thanks.

**Review by ritu9898 — APPROVED** [review]

**Review by dev-prabhat — APPROVED** [review]
Code is clean and understandable  and the custom slider is super cool you can merge

**Pull request merged**